dotnet/6.0/runtime/Dockerfile

```diff
@@ -0,0 +1,28 @@
+ARG newrelic_agent_version=9.0.0
```

Comment:
looks like we need to update the new relic agent too as only 9.2.0 supports .net6

https://github.com/newrelic/newrelic-dotnet-agent/releases/tag/v9.2.0

You can update that here: https://github.com/clickviewapp/dockerfiles/blob/master/dotnet/_templates/Dockerfile.template#L1

Reply:
Done